lib/src/Uncertainty/Algorithm/Sensitivity/openturns/HSICUStat.hxx

```diff
@@ -0,0 +1,56 @@
+//                                               -*- C++ -*-
+/**
+ * @brief HSICUStat implements the computation of an unbiased HSIC index.
+ *
+ *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_HSICUSTAT_HXX
+#define OPENTURNS_HSICUSTAT_HXX
+
+#include "openturns/HSICStatImplementation.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class HSICUStat
+ *
+ */
+class OT_API HSICUStat: public HSICStatImplementation
+{
+  CLASSNAME
+public:
+
+  /** Default constructor */
+  HSICUStat();
+
+  /** Virtual constructor */
+  HSICUStat* clone() const override;
+
+public:
+
+  /** Compute the HSIC index for one marginal*/
+  virtual Scalar computeHSICIndex(const Sample & inSample, const Sample & outSample, const CovarianceModel & inCovariance, const CovarianceModel & outCovariance, const SquareMatrix & weightMatrix) const override;
```

Comment:
this line is a bit long

Reply:
@jschueller 
We may add the `--max-code-length=xx` to the indent.sh script (with xx=50..200).